.ci/e2e_integration_test/start-e2e.ps1

```diff
@@ -97,5 +97,5 @@ Write-Host "--------------------------------------------------------------------
 Write-Host "Running E2E integration tests..." -ForegroundColor Green
 Write-Host "-----------------------------------------------------------------------------`n" -ForegroundColor Green
 $env:CORE_TOOLS_EXE_PATH = "$funcExePath"
-pytest --junitxml=e2e-integration-test-report.xml --cov=./azure_functions_worker --cov-branch --cov-append tests/endtoend --cov-report xml --cov-report html
+pytest --junitxml=e2e-integration-test-report.xml --reruns 4 --cov=./azure_functions_worker --cov-branch --cov-append tests/endtoend --cov-report xml --cov-report html
```

Comment:
Is the codecoverage thing used anywhere? Or posted?
If not, then lets clean that up as well

Reply:
You can reintroduce something like below to show the coverage back.

https://github.com/Azure/azure-functions-python-worker/blob/8031ffec97c7b060d2146431b1e6b2df7ed2956c/azure-pipelines.yml#L55